examples/ng-upgrade/README.md

```diff
@@ -1,8 +1,8 @@
 # ng-upgrade example
 
-This example is a stand alone Angular 1.x / Angular 2 hybrid application and
+This example is a stand alone Angular 1.x / Angular hybrid application and
```

Comment:
Same here, when comparing two versions of Angular I would use "Angular 1 / Angular 2" like "Chrome 54 / Chrome 55" like Google is doing for their browser.